tests/test_apis.py

```diff
@@ -0,0 +1,41 @@
+from fstat import app
+
+import unittest
+import json
+from datetime import datetime, timedelta
+
+
+class TestRestfulApis(unittest.TestCase):
+    """ Integration tests for the rest apis """
+
+    def setUp(self):
+        self.tester = app.test_client()
+
+    def test_get_failure_route(self):
+        """ Tests /api/failures route """
+        endpoint = '/api/failures'
+        # HACK: spliting string by T to get only date
+        query_string_params = {
+            'start_date': str(datetime.today() - timedelta(days=7)).split(" "),
+            'end_date': str(datetime.today()).split(" "),
+            'branch': 'master'
+        }
+
+        response = self.tester.get(endpoint, query_string=query_string_params)
+        assert response.status_code == 200
+        data = json.loads(response.data)
+        assert len(data['response']) > 0
```

Comment:
Ohh looks I when I wrote the tests I didn't it hurry so I might have forgot to validate the response. I will add that